test/type.query.test.js

```diff
@@ -498,7 +498,7 @@ describe('type: query', () => {
             // check number of API calls
             assert.equal(
                 testUtils.getAPIHistoryLength(),
-                35,
+                57,
```

Comment:
similar to above: that seems a bit excessive? have you checked whats causing the massive increase?

Reply:
Yes, 14 new  automations , automation is a dependent type for query so it is retrieved, 2 requests per each so now should be 35+28=63